src/playground-ide.ts

```diff
@@ -188,17 +190,31 @@ export class PlaygroundIde extends LitElement {
   @property({type: Boolean})
   resizable = false;
 
+  /**
+   * Get or set the array of project files.
+   *
+   * Files set through this property always take precedence over `projectSrc`
+   * and slotted children.
+   */
+  @property({type: Object})
```

Comment:
is `{type: Object}` intentional? If you're not deserializing from JSON it doesn't buy you anything over `attribute: false`, but you might be deserializing from JSON here...

Reply:
Switched to `attribute: false`.